src/PiWeb.Import.Sdk/PiWeb.Import.Sdk.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   
   <PropertyGroup Label="NuGet package specifications">
-    <Version>0.1.0</Version>
+    <Version>1.0.0-beta0037-66</Version>

```

Comment:
Please undo

Reply:
Oh, that was not supposed to be in here.